presto-hive/src/main/java/com/facebook/presto/hive/HdfsConfigurationUpdater.java

```diff
@@ -93,6 +95,8 @@ public HdfsConfigurationUpdater(HiveClientConfig hiveClientConfig, HiveS3Config
         this.fileSystemMaxCacheSize = hiveClientConfig.getFileSystemMaxCacheSize();
 
         this.s3AwsAccessKey = s3Config.getS3AwsAccessKey();
+        this.wasbAccessKey = wasbConfig.getWasbAccessKey();
```

Comment:
You're cutting the S3 'paragraph' into two by placing the two lines here. I suggest starting another code 'paragraph' just for blob either after or before the S3 'paragraph'.